docs/public/specifications/tests/Basic-TP/basic01.md

```diff
@@ -89,169 +89,167 @@ DNS queries follow, unless otherwise specified below, what is specified for
    2. Exit the test case.
 
 2. Create [DNS queries][DNS Query]:
-   1. Query type SOA and query name *Child Zone* ("SOA Child Query").
-   2. Query type DNAME and query name *Child Zone* ("DNAME Query").
+   1. Query type DNAME and query name *Child Zone* ("DNAME Child Query").
 
 3. Create the following empty sets:
```

Comment:
For error reporting’s sake, I’d suggest having sets of (name server name, name server IP, zone) tuples instead of merely (name server IP, zone). Having the name server’s name in addition to its IP address in the error messages makes remediating problems much easier.

Reply:
Agree. We should move from ns_ip to ns. Updated.